From e1fa52380f55fc5a41244f888a333dd8e7095c14 Mon Sep 17 00:00:00 2001
From: Thomas Cahuzac <thomasx.cahuzac@intel.com>
Date: Tue, 5 Feb 2013 18:49:44 +0100
Subject: Context awareness support

BZ: 85264
[CLEAN]

Add constants needed by the Context Awareness feature.

Category: feature differentiation
Domain: AudioComms-ContextAwareness
Origin: internal
Upstream-Candidate: no, proprietary
Change-Id: I61e0f6b2196f6f02669f762299ad7af53dd77afc
Orig-Change-Id: I0a0d89750dd26b783070b59762c3b3a62948207c
Signed-off-by: Thomas Cahuzac <thomasx.cahuzac@intel.com>
---
 include/hardware/audio.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 7db5415..1aa2de2 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -112,6 +112,11 @@ __BEGIN_DECLS
 #define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_ON "true"
 #define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_OFF "false"
 
+/* Context awareness */
+#define AUDIO_PARAMETER_KEY_CONTEXT_AWARENESS_STATUS "context_awareness_status"
+#define AUDIO_PARAMETER_VALUE_CONTEXT_AWARENESS_ON "on"
+#define AUDIO_PARAMETER_VALUE_CONTEXT_AWARENESS_OFF "off"
+
 /**
  *  audio stream parameters
  */
-- 
1.7.4.1

